soh/src/overlays/actors/ovl_Item_Etcetera/z_item_etcetera.c

```diff
@@ -242,6 +246,10 @@ void ItemEtcetera_Draw(Actor* thisx, GlobalContext* globalCtx) {
             this->giDrawId = GetItemModelFromId(GetRandomizedItemIdFromKnownCheck(RC_LH_SUN, GI_ARROW_FIRE));
```

Comment:
looks like we forgot to test the fire arrow check

Reply:
Done